docs/platforms/python/integrations/rust_tracing/index.mdx

```diff
@@ -0,0 +1,187 @@
+        .init();
+}
+
+#[pymodule]
+fn my_rust_extension(m: &Bound<'_, PyModule>) -> PyResult<()> {
+    m.add_function(wrap_pyfunction!(initialize_tracing, m)?)?;
+
+    Ok(())
+}
+```
+
+### Python
+
+Create an instance of `RustTracingIntegration` and add it to your list of integrations when initializing the Sentry SDK.
+
+<OnboardingOptionButtons
+  options={[
+    'error-monitoring',
+    'performance',
+    'profiling',
+  ]}
+/>
+
+```python {filename:main.py} {"onboardingOptions": {"performance": "9-11", "profiling": "12-15"}}
+import sentry_sdk
+from sentry_sdk.integrations.rust_tracing import RustTracingIntegration
+
+import my_rust_extension
+
+async def main():
+    sentry_sdk.init(
+        dsn="___PUBLIC_DSN___",
+        # Set traces_sample_rate to 1.0 to capture 100%
+        # of transactions for tracing.
+        traces_sample_rate=1.0,
+        # Set profiles_sample_rate to 1.0 to profile 100%
+        # of sampled transactions.
+        # We recommend adjusting this value in production.
+        profiles_sample_rate=1.0,
+        integrations=[
+            RustTracingIntegration(
+                "my_rust_extension",
+                my_rust_extension.initialize_tracing,
+                include_tracing_fields=True,
+            ),
+        ],
+    )
+
+    # your code goes here.
+    ...
+
+asyncio.run(main())
+```
+
+
+## Verify
+
+A simple way to check if the integration is hooked up correctly is to set a custom `event_type_mapping` and `span_filter` that prints or logs `tracing` event metadata and then call a Python function that uses your Rust native extension.
+```python
+from sentry_sdk.integrations.rust_tracing import (
+    default_event_type_mapping,
+    default_span_filter,
+    EventTypeMapping,
+    RustTracingIntegration,
+)
+
+import my_rust_extension
+
+def custom_event_type_mapping(metadata: dict) -> EventTypeMapping:
+    print(metadata)
+    return default_event_type_mapping(metadata)
+
+def custom_span_filter(metadata: dict) -> bool:
+    print(metadata)
+    return default_span_filter(metadata)
+
+sentry_sdk.init(
+    # ...
+    integrations=[
+        RustTracingIntegration(
+            "my_rust_extension",
+            my_rust_extension.initialize_tracing,
+            event_type_mapping=custom_event_type_mapping,
+            span_filter=custom_span_filter,
+        ),
+    ],
+)
+```
+
+To see the results on [sentry.io](https://sentry.io), go to the Traces section for your project and search for a Python span that calls a function from your Rust native extension. If the integration is working and the Rust function is instrumented with the Rust `tracing` framework, then the Python span will have a Rust child, and the Rust child may have a whole tree of Rust spans beneath it.
+
+The `pyo3-python-tracing-subscriber` crate has [a working example of a Sentry integration](https://github.com/codecov/pyo3-python-tracing-subscriber/tree/main/demo).
```

Comment:
ahh great catch! will update. i drafted this before moving the repo from `codecov` to `getsentry`, and it doesn't 404 for me because i still can access the old private copy :v